courses/free-courses-te.md

```diff
@@ -0,0 +1,38 @@
+## Index
+
+* [C](https://youtu.be/HdvRHC5TiwE)
+* [C++](https://youtu.be/uZBXKmQH5u8)
+* [JavaScript](https://youtu.be/pDdgpwRA_lI)
+* [jQuery](https://youtu.be/tggCrJGmzCA)
+* [PHP](https://youtu.be/Z3Y8z09dmF4)
+* [Python](https://youtu.be/LajgX-9nNB8)
+
+
+### C
+
+* [C](https://youtu.be/HdvRHC5TiwE)
```

Comment:
Please add more information about the video instead of just name of the language